Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -232,7 +232,8 @@ void ProcessThrottler::didConnectToProcess(ProcessID pid)
 
     m_processIdentifier = pid;
     updateThrottleStateNow();
-    RELEASE_ASSERT(m_assertion);
+    if (m_state != ProcessThrottleState::Suspended || m_shouldTakeSuspendedAssertion)
+        RELEASE_ASSERT(m_assertion);
```

Comment:
This is OK but I think I would have written:
```
RELEASE_ASSERT(m_assertion || (m_state == ProcessThrottleState::Suspended && ! m_shouldTakeSuspendedAssertion));
```

Reply:
I like that better too. Changed.